src/_data/authors.json

```diff
@@ -1,148 +1,154 @@
-			"photo": "https://avatars0.githubusercontent.com/u/498678?s=460&v=4",
-			"biography": "Web Developer and forever a student."
-		},
-		{
-			"name": "Monika Piotrowicz",
-			"website": "https://twitter.com/monsika",
-			"photo": "https://avatars1.githubusercontent.com/u/882875?s=460&u=ee95409a6edcde041c98ffa9d161767c0eef35b9&v=4",
-			"biography": "Director of International UX at Shopify, leading a globally distributed team, localizing a product."
-		},
-		{
-			"name": "Nat Tarnoff",
-			"website": "http://tarnoff.info/",
-			"photo": "https://pbs.twimg.com/profile_images/855580021951537152/bS14U2d5_400x400.jpg",
-			"biography": "Founder of MadHackerHaus and Rolled Up Sleeves, LLC. Advocate for accessibility in developing the internet and mobile technology."
-		},
-		{
-			"name": "Rachel R. Vasquez",
-			"website": "http://rachievee.com",
-			"photo": "https://avatars0.githubusercontent.com/u/10246844?s=460&v=4",
-			"biography": "Senior WordPress Developer."
-		},
-		{
-			"name": "Scott O'Hara",
-			"website": "https://www.scottohara.me/",
-			"photo": "https://avatars2.githubusercontent.com/u/4152514?s=460&v=4",
-			"biography": "Scott O'Hara is a designer and developer based outside of Boston MA. He writes code and cares a ton about building great, accessible user experiences for everyone."
-		},
-		{
-			"name": "Susan Robertson",
-			"website": "https://www.susanjeanrobertson.com/",
-			"photo": "https://pbs.twimg.com/profile_images/753022638725804032/x1JCSj_l_400x400.jpg",
-			"biography": "Hi. I'm Susan Robertson, a design implementer. I spend my days thinking about HTML, CSS, accessibility and design systems."
-		},
-		{
-			"name": "Scott Vinkle",
-			"website": "https://scottvinkle.me/",
-			"photo": "https://avatars1.githubusercontent.com/u/1392632?s=460&u=703b41be24a5fde23738489682f9cf5d642dd736&v=4",
-			"biography": "Hi, I'm Scott. Web accessibility consultant and front end engineer from Toronto, Canada."
-		},
-		{
-			"name": "Thomas Sjögren",
-			"website": "https://github.com/thomassjogren",
-			"photo": "https://avatars3.githubusercontent.com/u/3251254?s=460&v=4",
-			"biography": "UX focused front-end developer with passion for behaviour-driven design."
-		},
-		{
-			"name": "Luis Garcia",
-			"website": "https://www.garcialo.com/",
-			"photo": "https://pbs.twimg.com/profile_images/1181989762275528705/4ou0JnLG_400x400.jpg",
-			"biography": "Father, accessibility evangelist, video gamer, proof checker, and solver of problems."
-		},
-		{
-			"name": "Duncan Jimbo",
-			"website": "https://simplydano.com",
-			"photo": "https://avatars1.githubusercontent.com/u/1507672?s=460&u=bc8cff324d5739f946ad5dd15b5857e653de7ab8&v=4",
-			"biography": "Howdy! I'm Duncan, a front-end developer focused on design systems."
-		},
-		{
-			"name": "…and maybe you!",
-			"website": "/contribute/",
-			"photo": "/img/logos/maintainer.png",
-			"biography": "Got an idea for an post? Get in touch!"
-		}
-	]
-}
+{

```

Comment:
It appears you have an autoformatter in your code editor that is updating this file. As your author info will be added in https://github.com/a11yproject/a11yproject.com/pull/1108, it will not be necessary to include this file, or `package-lock.json` in your PR.